spyder/plugins/editor/widgets/tests/test_hints_and_calltips.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+#
+"""Tests for editor calltips and hover hints tooltips."""
+
+# Standard library imports
+import sys
+
+# Third party imports
+from qtpy.QtCore import Qt, QPoint
+from qtpy.QtGui import QTextCursor
+import pytest
+
+# Constants
+PY2 = sys.version[0] == '2'
+TEST_SIG = 'some_function(hello=None)'
+TEST_DOCSTRING = "This is the test docstring."
+TEST_TEXT = """'''Testing something'''
+def {SIG}:
+    '''{DOC}'''
+
+
+some_function""".format(SIG=TEST_SIG, DOC=TEST_DOCSTRING)
+
+
+@pytest.mark.slow
```

Comment:
Please also mark this test as `second` (as it's done in `test_warnings`).

Reply:
Will do